status.sh

```diff
@@ -23,30 +23,33 @@ check(){
     name="${3}"
     expectedcode="${4}"
 
+    # Sanitize the name by replacing / with _ so it will not be treated as part of file path.
+    tmp_name="$(echo "${name}" | sed 's,/,_,' )"
```

Comment:
can we call this sanitised_name?

Reply:
Sure thing, was just thinking it as a template name

Reply:
Updated

Reply:
Commit didn't support backtick `. Renamed `tmp_name` to `sanitised_name`